### ISSUE TITLE
The Cropping2D layer does not support variable input size (None)

### ISSUE THREAD

**Issue by Sushobhan04** [keunwoochoi/keras_cropping_layer#2]
The Cropping2D layer does not support variable input size (None) in convolutional layers. It throws the following error:

 unsupported operand type(s) for -: 'NoneType' and 'int'

**Comment by keunwoochoi** [comment]
This layer is merged and updated in Keras. Is it a question regarding it?

**Comment by Sushobhan04** [comment]
yes. I tried using it on a convolutional net, but it falied to work if I do not specify the input dimension. In CNNs, its particularly helpful as the networks can then run on size independent input.

**Comment by keunwoochoi** [comment]
I see. But this should be discussed in Keras issue board. There have been some changes, which means there are people keen on cropping layers. So there would be some responses.
